src/searchModal/scss/searchModal.scss

```diff
@@ -138,6 +127,27 @@ $classTreeVariables: (
         overflow: auto;
     }
 
+    .content-toolbar {
+        height: 2rem;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        align-items: stretch;
```

Comment:
Are `align-items` and `align-content` really needed?

Reply:
You are right, these additional rules are not needed. As we only have one link so far in the toolbar, no need to make the stylesheet heavier.